modules/admin/proc-install-operator.adoc

```diff
@@ -19,23 +23,38 @@ Containers are available for the following CPU architectures:
 * You are logged in as an administrator on the {ocp-short} web console.
 * You have configured the appropriate roles and permissions within your project to create an application. For more information, see the link:https://docs.openshift.com/container-platform/{ocp-version}/applications/index.html[Red Hat OpenShift documentation on Building applications].
 
-.Procedure
+[NOTE]
+====
+For enhanced security, deploy the {product} Operator in a dedicated default namespace such as `rhdh-operator`. The cluster administrator can restrict other users' access to the Operator resources through role bindings or cluster role bindings.
+====
 
-. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators* > *OperatorHub*.
+.Procedure
 
+. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators* -> *OperatorHub*.
 . In the *Filter by keyword* box, enter {product-short} and click the *{product}* Operator card.
-
 . On the *{product} Operator* page, click *Install*.
 
-. On the *Install Operator* page, click *Install*.
+. On the *Install Operator* page, use the *Update channel* drop-down menu to choose an update channel:
+** The *fast* channel provides y-stream (x.y) and z-stream (x.y.z) updates, for example, updating from version 1.1 to 1.2, or from 1.1.0 to 1.1.1.
++
+[IMPORTANT]
+====
+The `fast` channel includes all updates, any may introduce unexpected changes in your {product} deployment. Check the release notes for details about any potentially breaking changes.
```

Comment:
```suggestion
The `fast` channel includes all of the updates available for a particular version. Any update might introduce unexpected changes in your {product} deployment. Check the release notes for details about any potentially breaking changes.
```
You can take or leave the changes to the first sentence, but I do recommend breaking this into 2 sentences and uses "might" instead of "may".